styles/pronouns.cc/catppuccin.user.css

```
@@ -0,0 +1,232 @@
+/* ==UserStyle==
+@name pronouns.cc Catppuccin
+@namespace github.com/catppuccin/userstyles/styles/pronouns.cc
+@homepageURL https://github.com/catppuccin/userstyles/tree/main/styles/pronouns.cc
+@version 0.0.1
+@updateURL https://github.com/catppuccin/userstyles/raw/main/styles/pronouns.cc/catppuccin.user.css
+@supportURL https://github.com/catppuccin/userstyles/issues?q=is%3Aopen+is%3Aissue+label%3Apronouns.cc
+@description Soothing pastel theme for pronouns.cc
+@author Catppuccin
+@license MIT
+
+@preprocessor less
+@var select lightFlavor "Light Flavor" ["latte:Latte*", "frappe:Frappé", "macchiato:Macchiato", "mocha:Mocha"]
+@var select darkFlavor "Dark Flavor" ["latte:Latte", "frappe:Frappé", "macchiato:Macchiato", "mocha:Mocha*"]
+@var select accentColor "Accent" ["rosewater:Rosewater", "flamingo:Flamingo", "pink:Pink", "mauve:Mauve*", "red:Red", "maroon:Maroon", "peach:Peach", "yellow:Yellow", "green:Green", "teal:Teal", "blue:Blue", "sapphire:Sapphire", "sky:Sky", "lavender:Lavender", "subtext0:Gray"]
+==/UserStyle== */
+
+@-moz-document domain('pronouns.cc') {
+  :root[data-bs-theme="dark"] {
+    #catppuccin(@darkFlavor, @accentColor);
+    .bg-dark {
+      --bs-bg-opacity: 1;
+      background-color: rgba(
+        var(--bs-lightt-rgb),
+        var(--bs-bg-opacity)
+      ) !important;
+  }
+  :root[data-bs-theme="light"] {
+    #catppuccin(@lightFlavor, @accentColor);
+    .bg-light {
+      --bs-bg-opacity: 1;
+      background-color: rgba(
+        var(--bs-darkk-rgb),
+        var(--bs-bg-opacity)
+      ) !important;
+    }
+
+  }
```

```suggestion
  :root[data-bs-theme="dark"] {
    #catppuccin(@darkFlavor, @accentColor);
    .bg-dark {
      --bs-bg-opacity: 1;
      background-color: rgba(
        var(--bs-lightt-rgb),
        var(--bs-bg-opacity)
      ) !important;
    }
  }
  :root[data-bs-theme="light"] {
    #catppuccin(@lightFlavor, @accentColor);
    .bg-light {
      --bs-bg-opacity: 1;
      background-color: rgba(
        var(--bs-darkk-rgb),
        var(--bs-bg-opacity)
      ) !important;
    }
  }
```

You need to be testing changes you make to the userstyle. Light themes didn't load because of this missing bracket - when you change something in a specific area of the userstyle, check to see it (still) works!